### PR TITLE
TAO-8586 - initial commit

### PR DESCRIPTION
**Pull request by yuryfilippovich**

### CODE REVIEW

File: controller/TaoSyncServer.php [hunk 1]
```diff
@@ -0,0 +1,52 @@
+<?php
```
Comment (ks16) [inline controller/TaoSyncServer.php]: We don't need this controller at the moment

File: locales/en-US/messages.po [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+"Language: en-US\n"
+"sourceLanguage: en-US\n"
+"targetLanguage: en-US\n"
+"Content-Type: text/plain; charset=utf-8\n"
+"Content-Transfer-Encoding: 8bit\n"
+"X-Poedit-Basepath: ../../\n"
+"X-Poedit-KeywordsList: __\n"
+"X-Poedit-SearchPath-0: .\n"
+
+msgid "Hello World"
```
Comment (ks16) [inline locales/en-US/messages.po]: Remove unnecessary translations

File: manifest.php [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+    'name' => 'taoSyncServer',
+    'label' => 'Tao Sync Central Server',
+    'description' => 'TAO central server for synchronisation.',
+    'license' => 'GPL-2.0',
+    'version' => '0.0.1',
+    'author' => 'Open Assessment Technologies SA',
+    'requires' => [],
+    'managementRole' => 'http://www.tao.lu/Ontologies/generis.rdf#taoSyncServerManager',
+    'acl' => [
+        ['grant', 'http://www.tao.lu/Ontologies/generis.rdf#taoSyncServerManager', ['ext'=>'taoSyncServer']],
```
Comment (ks16) [inline manifest.php]: We can use constant here for role name - https://github.com/oat-sa/extension-tao-sync/blob/master/manifest.php#L52

File: views/templates/TaoSyncServer/templateExample.tpl [hunk 4]
```diff
@@ -0,0 +1,3 @@
+<div class="main-container tao-scope">
```
Comment (ks16) [inline views/templates/TaoSyncServer/templateExample.tpl]: Remove unnecessary template

**Review by ks16 — CHANGES_REQUESTED** [review]
Could you please create Updater script as well?

File: composer.json [hunk 5]
```diff
@@ -0,0 +1,31 @@
+{
+
+    "name": "oat/taoSyncServer",
```
Comment (ks16) [inline composer.json]: Composer package should be as repository name on GitHub - https://github.com/oat-sa/extension-tao-sync/blob/master/composer.json#L2

**Review by ks16 — CHANGES_REQUESTED** [review]

**Review by ks16 — CHANGES_REQUESTED** [review]
Incorrect package name in composer.json

File: composer.json (hunk 5 above)
Comment (yuryfilippovich) [reply composer.json]: fixed

**Pull request merged**